evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -136,7 +133,7 @@ def next_batch(self):
         if self._batch_number == 0:
             next_batch = [
                 pipeline.new(
-                    parameters=self._combine_parameters(pipeline, {}),
+                    parameters={**self._transform_parameters(pipeline, {})},
```

Comment:
I think in all three lines in this diff we can just do `parameters=self._transform_parameters(pipeline, ...`